package.json

```diff
@@ -151,6 +151,11 @@
           "default": "green",
           "markdownDescription": "Background color of the decoration. Use rgba() and define transparent background colors to play well with other decorations. Example: green"
         },
+        "twitchHighlighter.highlightFontColor": {
+          "type": "string",
+          "default": "white",
+          "markdownDescription": "Font color of the decoration. Use rgba() and define transparent background colors to play well with other decorations. Example: white"
```

Comment:
I thought this was already added in another PR that was merged. Wasn't it?

Reply:
Yes, I believe there was a merge conflict in my local repo with the `package.json` that I fixed locally and this setting was missing so I fixed that up.

Reply:
Wonder why it's showing in the diff still. Seems odd, right?

Reply:
It looks like the configuration section in `vnext` is missing the setting...

Reply:
The `extension.ts` is referencing the setting however

https://github.com/clarkio/vscode-twitch-highlighter/blob/33631dfd377fc10d2cbf2a3b76da53386e878780/src/extension.ts#L508

I believe this might have been caused by that merge conflict we had on Friday.

Reply:
Alright, no worries then. Was just curious.